contracts/Bank.sol

```diff
@@ -214,18 +214,45 @@ contract Bank is Constant, Initializable{
     }
 
     /**
-     * Get the borrowing interest rate Borrowing interest rate.
+     * Get the borrowing interest rate.
      * @param _token token address
      * @return the borrow rate for the current block
      */
     function getBorrowRatePerBlock(address _token) public view returns(uint) {
-        if(!globalConfig.tokenInfoRegistry().isSupportedOnCompound(_token))
-        // If the token is NOT supported by the third party, borrowing rate = 3% + U * 15%.
-            return getCapitalUtilizationRatio(_token).mul(globalConfig.rateCurveSlope()).div(INT_UNIT).add(globalConfig.rateCurveConstant()).div(BLOCKS_PER_YEAR);
-
-        // if the token is suppored in third party, borrowing rate = Compound Supply Rate * 0.4 + Compound Borrow Rate * 0.6
-        return (compoundPool[_token].depositRatePerBlock).mul(globalConfig.compoundSupplyRateWeights()).
-            add((compoundPool[_token].borrowRatePerBlock).mul(globalConfig.compoundBorrowRateWeights())).div(10);
+        uint256 capitalUtilizationRatio = getCapitalUtilizationRatio(_token);
+        // rateCurveConstant = <'3 * (10)^16'_rateCurveConstant_configurable>
+        uint256 rateCurveConstant = globalConfig.rateCurveConstant();
+        // compoundSupply = Compound Supply Rate * <'0.4'_supplyRateWeights_configurable>
+        uint256 compoundSupply = compoundPool[_token].depositRatePerBlock.mul(globalConfig.compoundSupplyRateWeights());
+        // compoundBorrow = Compound Borrow Rate * <'0.6'_borrowRateWeights_configurable>
+        uint256 compoundBorrow = compoundPool[_token].borrowRatePerBlock.mul(globalConfig.compoundBorrowRateWeights());
+        // nonUtilizedCapRatio = (1 - U) // Non utilized capital ratio
+        uint256 nonUtilizedCapRatio = INT_UNIT.sub(capitalUtilizationRatio);
+
+        bool isSupportedOnCompound = globalConfig.tokenInfoRegistry().isSupportedOnCompound(_token);
+        if(isSupportedOnCompound) {
+            uint256 compoundSupplyOrBorrow = compoundSupply.add(compoundBorrow).div(10);
```

Comment:
the name of the variable looks like we either have `compoundSupply` or `compoundBorrow`. But this variable has both. The variable name should be `compoundSupplyPlusBorrow`

Reply:
I had previously named it compoundSupplyAndBorrow but changed it because we're doing an `add` and `AND` would imply multiplication.
So just that so it's not confusing, I did that

Reply:
renamed to `compoundSupplyPlusBorrow`